### PR TITLE
Update to 3.2a

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{% set version = "3.1" %}
-{% set sha256 = "979bf38db2c36193de49149aaea5c540d18e01ccc27cf76e2aff5606bd186722" %}
+{% set version = "3.2a" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:1]: How is Conda going to handle this version number?

Comment (dbast) [reply recipe/meta.yaml:1]: According to https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html conda is following https://www.python.org/dev/peps/pep-0440/#version-scheme `[N!]N(.N)*[{a|b|rc}N][.postN][.devN]` ... that means 3.2a should be fine.

**Comment by dbast** [comment]
seems that still some patching is required to get that building for osx-arm64

**Comment by dbast** [comment]
all green :)

**Comment by dbast** [comment]
ping @conda-forge/tmux

**Comment by dbast** [comment]
ping

**Comment by XhstormR** [comment]
ping

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:1]: Is this an alpha? If so, we should be following [CFEP 5]( https://github.com/conda-forge/cfep/blob/main/cfep-05.md )

**Comment by XhstormR** [comment]
Has anyone noticed that this pr can be merged?

**Comment by jakirkham** [comment]
There is review discussion happening above

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:1]: @dbast?

Comment (dbast) [reply recipe/meta.yaml:1]: tmux 3.2a is bugfix release of the 3.2 version (not an alpha version)... see https://github.com/tmux/tmux/releases

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Pull request merged**